presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -345,10 +356,11 @@ private void printQueryInfo(QueryStatusInfo results)
             reprintLine(stagesHeader);
 
             printStageTree(stats.getRootStage(), "", new AtomicInteger());
+            renderWarningsToConsole(results.getWarnings(), warningCount);
         }
         else {
             // Query 31 [S] i[2.7M 67.3MB 62.7MBps] o[35 6.1KB 1KBps] splits[252/16/380]
-            String querySummary = String.format("Query %s [%s] i[%s %s %s] o[%s %s %s] splits[%,d/%,d/%,d]",
```

Comment:
Add a separate commit for this static import change (since it's an unrelated refactoring)